docs/web/sdk-component-reference.mdx

```diff
@@ -0,0 +1,384 @@
+---
+title: Component Reference
+description: Geocortex Web - Reference for custom components in Geocortex Web
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import UseCaseCard from "../../src/components/UseCaseCard";
+import UseCaseContainer from "../../src/components/UseCaseContainer";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+The following is an example of a custom component that has been created using the Geocortex Web SDK. In general, Geocortex Web components are functional React components extended with a few new behaviors and patterns.
+
+```tsx
```

Comment:
Should this have a file name do you think?

```suggestion
```tsx title="src/components/ExampleComponent/ExampleComponent.tsx"
```

Reply:
The reason I didn't give it a filename is because it's not a complete example. I don't show the model code, registration code, layout, etc, that would be required to get this to run. Filenames, in my understanding, are for files belonging to complete copyable examples, and this example is not complete.

Reply:
Makes sense!